### ISSUE TITLE
LAUSD firewall sometimes blocks icon loading

### ISSUE THREAD

**Issue by glen3b** [aopell/SchoologyPlus#57]
Sometimes the LAUSD firewall will mess with TLS connections to the source of custom icons throughout Schoology Plus (although not always). This leads to image load failing. If this happens the alt text sometimes messes greatly with the UI and is overall a bad thing.

**Comment by glen3b** [comment]
Doesn't work on grades page.
I have logic to attempt a fallback but it's nonfunctional.

**Comment by glen3b** [comment]
The fix works by trying to load (via an `img`) the first image found per image domain (e.g. `image.flaticon.com`), and if that load fails then all images from that domain will be set to the fallback URL. I can't find a nice way to do this in CSS so I use a hacky JavaScript approach.
